CONTRIBUTING.md

```diff
@@ -55,6 +55,41 @@ docs(contributing-readme): add example of commit with subject line
 
 A commit hook makes sure the syntax is followed. Automated commit messages such as `Merge pull request` are handled.
 
+## Type generation
+This SDK support generating typescript types from the Cognite open api specification. The idea is to use openapi as a source of truth. Any incorrect or missing types should be fixed/added in the openapi spec instead of manually adjusting the generated types. This also helps to keep documentation up to date.
+
+The scripts prefix `codegen` are reserved for type generation. The core logic is found in `packages/codegen` with a deeper explenation.
+
+To quickly setup type generation for your service you may run the generate command with `--auto`:
+> Note: using `--auto` will setup a local versionfile and not the package global versionfile. You can later switch to the global versionfile once you have confirmed type generation works correctly for your service (you may also have to update your openapi spec).
```

Comment:
Wasn't that my suggestion earlier today - or am I misunderstanding something?

Reply:
Might be

Reply:
From what I can tell, it is. Perhaps you were thinking of something additional to my suggestion earlier? I've updated the PR description to reflect my suggestion so please take a look when you get back from vacay :sun_with_face:

Reply:
From what I can see, it is. Perhaps you were thinking about something additional?

I've updated the PR description and I'm in the progress of updating the code. Please take a look when you're back from vacay :sun_with_face:

Reply:
I see this content is no longer present. It seems now that you have to change the config to be able to regenerate with a custom path, e.g. for testing some change? I liked the way it was easy previously to generate types for local spec with least effort (e.g. while working on service contracts) - and I'm not sure if it needs to be coupled to a config as it would only be temporary?